README.md

```diff
@@ -203,14 +204,46 @@ Space      | Skip cutscenes, interact
 Ctrl + c   | Force Quit (from command-line)
 Ctrl + q   | Quit (ingame)
 
+## 5. Running The Longest Journey
+
+### 5.1. Required files ###
+
+You will need these elements from your game CDs or DVD :
+  * *TODO*
```

Comment:
Please complete this TODO. This files on the CDs are the same as with the digital releases. For the 4-CD releases they can be copied as is. For the 2-CD/DVD versions some files need to be extracted from the installer.

Reply:
I was planning to update the CD/DVD if/when I find the game but Unfortunately I think I misplaced it somewhere (and I think I only had 4 CD version that might even had one CD scratched), but I'll add info based on what you said (and probably can do some googling as well).